src/AcceptanceTests/Routing/When_subscribing_to_a_derived_polymorphic_event_with_forwarding_topology.cs

```diff
@@ -100,17 +100,16 @@ public Task Handle(EventWasRaisedSoStopProcessing message, IMessageHandlerContex
             }
         }
 
-        public interface BaseEvent : IEvent
+        public class BaseEvent : IEvent
```

Comment:
For the ASB transport, is there anything about how subscriptions for classes compared to interfaces? As long as there are no differences, then this change seems fine.

Reply:
No difference AFAIK.